.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
nit: move this to the top gitignore file. We most likely never want this.

Reply:
https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action#commit-tag-and-push-your-action-to-github

> This means you must include any package dependencies required to run the JavaScript code. You'll need to check in the toolkit core and github packages to your action's repository.

Github recommends committing the node_modules to the repo and as nasty as it sounds I agree given the `dist/index.js` in this PR is 30,000 lines of JS. Would also mean there's no need to build the index.js.

Reply:
> nit: move this to the top gitignore file. We most likely never want this.
This is the top gitignore (we didn't have one before). 

Ah good shout Jack, will do that.

Reply:
I've now spotted further down the article they suggest using ncc as you've done in this PR. Not sure which is the best of the choices tbh. 🤔

Reply:
Hmm okay. In all examples I've seen so far, they've used ncc. Maybe that's the least bad option after all?

Reply:
Yeah I think we stick with ncc after all. Sorry for the bother.